packages/gatsby-cli/src/issue/telemetry.js

```diff
@@ -0,0 +1,81 @@
+const path = require(`path`)
+const fs = require(`fs-extra`)
+
+const envInfo = require(`../env-info.js`)
+
+const mdCodeBlock = (content, lang) =>
+  `
+\`\`\`${lang}
+${content}
+\`\`\`
+  `.trim()
+
+module.exports = answers => {
+  if (!answers.envinfoConfirmation && !answers.filesConfirmation) {
+    return new Promise(resolve => resolve(``))
+  }
+
+  const promises = []
```

Comment:
nit: but I like to call this `let` since it's a signal that this will be mutated--even if it's not re-assigned.

Feel free to keep this as is though :)